pkg/personname/info.go

```diff
@@ -104,9 +119,9 @@ func (info Info) dcmRemoveNullStrings(groupStrings []string) []string {
 	// for.
 
 	// Start with the sliceOut point containing all three groups
-	sliceOut := 3
+	sliceOut := len(groupStrings)
```

Comment:
can we rename this to something like numGroups?

```suggestion
	numGroups := len(groupStrings)
```